apps/web/src/pages/activities/components/ActivityStatistics.tsx

```diff
@@ -0,0 +1,62 @@
+import { useQuery } from 'react-query';
+import styled from '@emotion/styled';
+import { getActivityStats } from '../../../api/activity';
+import { formatNumber } from '../../../utils';
+
+export function ActivityStatistics() {
+  const { data: activityStats, isLoading: loadingActivityStats } = useQuery<{
+    yearlySent: number;
+    monthlySent: number;
+    weeklySent: number;
+  }>('activityStats', getActivityStats);
+
+  return (
+    <>
+      {!loadingActivityStats ? (
+        <ContentWrapper>
+          <StatisticsBox>
+            <StyledNumber>{formatNumber(activityStats?.weeklySent, 0)}</StyledNumber>
+            <StyledText>This week</StyledText>
+          </StatisticsBox>
+          <StatisticsBox>
+            <StyledNumber>{formatNumber(activityStats?.monthlySent, 0)}</StyledNumber>
+            <StyledText>This month</StyledText>
+          </StatisticsBox>
+          <StatisticsBox>
+            <StyledNumber>{formatNumber(activityStats?.yearlySent, 0)}</StyledNumber>
+            <StyledText>This year</StyledText>
+          </StatisticsBox>
+        </ContentWrapper>
+      ) : null}
+    </>
+  );
+}
+
+const StatisticsBox = styled.div`
+  padding-right: 30px;
+  color: #828299; //colors.B60
```

Comment:
Is this one should be colors.B60?

Reply:
Yea it is 828299 is - colors.B60 however the colors gave me errors I left it there so I will look into it again later.